web/frontend/src/components/board.vue

```diff
@@ -3,7 +3,7 @@
 import { toast } from '../lib/toast';
 import { displayError } from '../lib/error';
 import { rcLogConditionally } from '../lib/log';
-import { Client, BoardClient, ServiceError } from '@viamrobotics/sdk';
+import { Client, BoardClient, type ServiceError } from '@viamrobotics/sdk';
```

Comment:
Upgrading Vite made it much more strict about the explicitness of `import type` when importing interfaces. I like that!